bin/ghcrawler.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const start = require('../src').start;
+const path = require('path');
+
+const args = require('argly').createParser({
+    '--config -c *': 'string'
+  })
+  .parse();
+
+Promise.resolve()
```

Comment:
For code hygiene and sanity, lets stick to one promise library in this project.  For now that is Q.  I'm happy to talk about the relative merits of other libs but that should be a separate effort.